packages/web/src/components/basic/ErrorBoundary.js

```diff
@@ -0,0 +1,64 @@
+import types from '@appbaseio/reactivecore/lib/utils/types';
+import { arrayOf, object } from 'prop-types';
+import React, { Component } from 'react';
+import { connect } from '../../utils';
+
+class ErrorBoundary extends Component {
+	constructor(props) {
+		super(props);
+		this.state = { error: null };
```

Comment:
@SavvyShah What is the need of maintaining the `error` state? You're reading `error` from props in render method.

Reply:
Sometimes we get `error` from `getDerivedStateFromError` when there are no network failures but other runtime errors.